packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -54,8 +54,13 @@ export default abstract class BaseAction {
 		}
 	}
 
-	protected async runSuiteCloudCommand(args: { [key: string]: string | string[] } = {}) {
-		const suiteCloudRunnerRunResult = await new SuiteCloudRunner(this.vsConsoleLogger, this.executionPath).run({
+	protected async runSuiteCloudCommand(args: { [key: string]: string | string[] } = {}, executionPath?: string) {
+		let executionPathToUse = this.executionPath;
+		if (executionPath !== undefined) {
+			executionPathToUse = executionPath;
+		}
```

Comment:
I think we can use here a ternary operator to decide `executionPahtToUse`, and I would like to rename the `executionPath` parameter to `externalExecutionPath` or `outerExecutionPath` to make clearer its propose and easier to differentiate from `this.executionPath`.

Reply:
Maybe `alternativeExecutionPath` could be an option but I'm not sure about it because it will be used if present, instead of an alternative if `this.excutionPath` is missing.

Reply:
Updated to make use of otherExecutionPath.